.ci/run-benchmark.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+set -e
+
+# check if jar path is provided
+JAR_PATH=$1
+if [ -z "$JAR_PATH" ]; then
+    echo "Usage: $0 <target jar path>"
+    exit 1
+fi
+
+# Sample project path
+SAMPLE_PROJECT="./.ci/jdk22"
+
+# generate benchmark config file
+generate_config() {
+  local CONFIG="<?xml version=\"1.0\"?>
+<!DOCTYPE module PUBLIC \"-//Puppy Crawl//DTD Check
+Configuration 1.3//EN\"
+\"http://checkstyle.sourceforge.net/dtds/configuration_1_3.dtd\">
+<module name=\"Checker\">
+  <module name=\"TreeWalker\"/>
```

Comment:
https://github.com/checkstyle/checkstyle/blob/b0c91b99d18f64eec419247010a7187120eedaca/src/main/java/com/puppycrawl/tools/checkstyle/TreeWalker.java#L153

We need at least one module.

Please make this config as static file.

Reply:
Shouldn't we choose a low cost module to be included? "Low cost" meaning a module which does extremely little work. Maybe just a quick visit and no begin/end.

Any module we add will be included in the performance test and will skew the results of calculating the grammar itself.

If we do this, we should add a warning comment too.

Reply:
Yes, if we only want to benchmark the grammar parsing, using a low cost module would definitely be better. I assume [`ConstantName`](https://checkstyle.sourceforge.io/checks/naming/constantname.html) is a good one, since it only runs regex on texts. Do you have recommandations on better modules in our case? I can even create an empty check just for testing purpose later if you think it would be more accurate and more helpful.

Reply:
I do not have a recommendation currently off the top of my head. I wouldn't consider regexp low cost, especially if it executes the regexp match, but if there is nothing lower than that is fine.

> create an empty check just for testing purpose 

I am fine with this, but I would look for approval from @romani .

Reply:
Thanks for you advice, I will try `NodeCodeInFileCheck` for now. 😄

Reply:
Yes, https://checkstyle.org/checks/misc/nocodeinfile.html#NoCodeInFile is awesome candidate.

We also use https://github.com/checkstyle/contribution/blob/cdeb66c6bc719eb0cf27a3cc0aadafbe2c016a47/checkstyle-tester/my_check.xml#L23

Reply:
done, I have also added all necessary filters to prevent checkstyle stops with errors